playground/backend/internal/setup_tools/builder/setup_builder.go

```diff
@@ -31,73 +31,139 @@ const (
 	javaLogConfigFilePlaceholder = "{logConfigFile}"
 )
 
-// SetupExecutorBuilder return executor with set args for validator, preparator, compiler and runner
-func SetupExecutorBuilder(paths fs_tool.LifeCyclePaths, pipelineOptions string, sdkEnv *environment.BeamEnvs) (*executors.ExecutorBuilder, error) {
+// Validator return executor with set args for validator
+func Validator(paths *fs_tool.LifeCyclePaths, sdkEnv *environment.BeamEnvs) (*executors.ExecutorBuilder, error) {
```

Comment:
Why do you pass here the pointer? I think we don't want to change anything in `paths`, let's remove `*`, what do you think?

Reply:
Maybe, we can also remove `*` from `sdkEnv`?

Reply:
Let's do it at the next PR, it is not very critical right now